bodhi/server/templates/update.html

```diff
@@ -692,6 +692,23 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              % if request.registry.settings.get('ci.required', 'False') == 'True':
+              <div class="p-b-1">
+                <div>
+                  % if request.registry.settings.get('ci.url'):
```

Comment:
There is an ``% else`` so it won't be empty.

Why would settings not be defined? (it is used elsewhere in the template, so I guess we would have more troubles if it isn't)